extra-dev/packages/coq-equations/coq-equations.dev+HoTT/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+authors: [ "Matthieu Sozeau <matthieu.sozeau@inria.fr>" "Cyprien Mangin <cyprien.mangin@m4x.org>" ]
+maintainer: "matthieu.sozeau@inria.fr"
+homepage: "https://mattam82.github.io/Coq-Equations"
+dev-repo: "git+https://github.com/mattam82/Coq-Equations.git"
+bug-reports: "https://github.com/mattam82/Coq-Equations/issues"
+license: "LGPL 2.1"
```

Comment:
```suggestion
license: "LGPL-2.1-or-later"
```
Is this "LGPL 2.1 or later" or "LGPL 2.1 only"? The suggestion is the SPDX identifier for the first case.

Reply:
Or later was intended indeed.